src/ophys_etl/modules/suite2p_registration/schemas.py

```diff
@@ -94,6 +94,29 @@ class Suite2PRegistrationInputSchema(argschema.ArgSchema):
         default=8,
         description="Maximum number of iterations to preform when creating a "
                     "reference image.")
+    remove_empty_frames = argschema.fields.Boolean(
+        required=False,
+        default=False,
+        allow_none=False,
+        description="Some movies arrive with otherwise quality data but have "
+                    "but contain a set of frames that are empty and contain "
+                    "pure noise. When processed, these frames tend to receive "
+                    "large random shifts that throw off motion border "
+                    "calculation. Turning on this setting on removes a "
+                    "defined set of frames from the movie before processing. "
+                    "The frames are still written however any shift estimated "
+                    "is removed and their shift is set to 0 to avoid large "
+                    "motion borders.")
+    trim_frames_start = argschema.fields.Int(
```

Comment:
Oh... Do we really want to expose these to the user? I figured these should only be auto-detected by your code.

Reply:
See above.

Reply:
I see. You said "overrides trim_frames_start/end".

I would feel better if there was an error associated with setting them all (which I guess would trigger if `auto_detect=True` and one of `trim_frames_start/end > 0`.

Reply:
Sure. I'll give the error option.